basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
+| [Date and time types](/data-type-date-and-time.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| [String types](/data-type-string.md)                                                 | Y     | Y   | Y   | Y   | Y   |
+| [JSON type](/data-type-json.md)                                                      | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Control flow functions](/functions-and-operators/control-flow-functions.md)         | Y     | Y   | Y   | Y   | Y   |
+| [String functions](/functions-and-operators/string-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Numeric functions and operators](/functions-and-operators/numeric-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Date and time functions](/functions-and-operators/date-and-time-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Bit functions and operators](/functions-and-operators/bit-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Cast functions and operators](/functions-and-operators/cast-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Encryption and compression functions](/functions-and-operators/encryption-and-compression-functions.md) | Y     | Y   | Y   | Y   | Y   |
+| [Information functions](/functions-and-operators/information-functions.md)           | Y     | Y   | Y   | Y   | Y   |
+| [JSON functions](/functions-and-operators/json-functions.md)                         | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Aggregation functions](/functions-and-operators/aggregate-group-by-functions.md)    | Y     | Y   | Y   | Y   | Y   |
+| [Window functions](/functions-and-operators/window-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Miscellaneous functions](/functions-and-operators/miscellaneous-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Operators](/functions-and-operators/operators.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| **Character sets**                                                                   | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [`utf8`](/character-set-and-collation.md)                                            | Y     | Y   | Y   | Y   | Y   |
+| [`utf8mb4`](/character-set-and-collation.md)                                         | Y     | Y   | Y   | Y   | Y   |
+| [`ascii`](/character-set-and-collation.md) [^1]                                      | Y     | Y   | Y   | Y   | Y   |
+| [`latin1`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| [`binary`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| **Collations**                                                                       | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
```

Comment:
ditto

Reply:
handled in https://github.com/pingcap/docs/pull/3960/commits/d55b7b555e1102efa83c3039eadb4f20676138b8